apps/src/lib/tools/jsdebugger/DebugConsole.jsx

```diff
@@ -149,10 +158,26 @@ export default connect(
           );
         } else if (this.props.isAttached) {
           try {
-            const result = this.props.evalInCurrentScope(input);
-            this.appendLog('< ' + String(result));
+            if (experiments.isEnabled('react-inspector')) {
+              let result = this.props.evalInCurrentScope(
+                input[0] === '{' && input[input.length - 1] === '}'
+                  ? `(${input})`
```

Comment:
NIT: Add a comment for why we wrap objects in parentheses here

Reply:
Will do!

Reply:
Did a bit of searching around, and it looks like the parentheses prevent the object from being interpreted as a block rather than as an object. Lots more on that here: https://stackoverflow.com/questions/9082110/defining-a-javascript-object-in-console

Reply:
Ah! Yes, that's a better comment to write! I'll change the comment.